source/include/JavaScriptRef.h

```diff
@@ -0,0 +1,42 @@
+/**
+
+ Copyright (c) 2018 National Instruments Corp.
+
+ This software is subject to the terms described in the LICENSE.TXT file
+
+ SDG
+ */
+
+/*! \file
+ \brief JavaScript refnum
+
+ JavaScript references are defined in the Params or Locals section of a VI,
+ and define a named, non-statically-bound reference to a JavaScript opaque object.
+ 
+ Example:
+ 
+ define(MyVI dv(.VirtualInstrument (
+ Locals: c(
+ e(JavaScriptRefNum  jsref)  // JavaScript ref that is not statically linked
+ ) ...
+ 
+ The actual Type of jsref is JavaScriptRefNum and stores a refnum (cookie) which
+ opaquely holds onto the javascript object.
```

Comment:
Please mention here that the refnum is managed on the JS side and does not actually use the Vireo RefNumManager.